vcf-dumper/vcf-dumper-lib/src/test/java/uk/ac/ebi/eva/vcfdump/VariantExporterControllerTest.java

```diff
@@ -171,7 +171,7 @@ public void testVcfHtsExportOneStudy()
 
         VariantExporterController controller = new VariantExporterController("hsapiens",
                                                                              VariantExporterTestDB.TEST_DB_NAME,
```

Comment:
Could this constant be renamed? There is a `SHEEP_TEST_DB_NAME` but it in unclear what this one contains. I thought it would be human data but the constant value is "VariantExporterTest" so still don't know.